docs/source/remote_access.rst

```diff
@@ -0,0 +1,80 @@
+
+Set up an SSH key pair
+^^^^^^^^^^^^^^^^^^^^^^
+If your local machine is a Mac, you can eliminate the need to enter a password every time you want to log in by using an SSH key pair. To do this, on your **local machine** navigate to your ``~/.ssh`` directory and enter the following command to generate a set of RSA keys: ``$ ssh-keygen -t rsa``. You will then be prompted to supply a filename and a password. For the file name I recommend ``id_rsa_hostalias`` and for the password I recommend it to be the same as your remote machine's password. If you use a Mac, the operating system can use its internal keychain to remember your password, meaning you won't need to type it in every time you log in!
+
+This command will generate 2 files:the one with the ``.pub`` extension is the "public key", the one without the ``.pub`` extension is the "private key". You keep your private key strictly on your local machine. You need to copy your public key to the remote machine you would like to use the key pair to log in to. First create the ``~/.ssh`` on your **remote machine**. Then from your **local machine** enter: ``$ scp ~/.ssh/id_rsa_hostalias.pub hostalias:~/.ssh/``. After copying the public key over to ``hostalias``, you need to create an ``authorized_keys`` file in your ``~/.ssh/`` on your **remote machine**.
+
+Now from a terminal window on your **local machine**, you can try logging in to ``hostalias`` with the following command: ``$ ssh -i ~/.ssh/id_rsa_hostalias hostalias``. This will prompt you for the password you specified upon creating your key pair using ``ssh-keygen``. To always make use of your private key when logging in to ``hostalias``, add the following to your ``config`` file on your **local machine**::
+
+   Hostname **hostname**
+      User **username**
+      LocalForward 3039 localhost:3039
+      IdentityFile **~/.ssh/id_rsa_hostname**
+      UseKeychain yes
+      AddKeysToAgent Yes
+
+You should not be able to log in simply by typing ``$ ssh hostalias``. Congratulations! 
+
+
+Edit remote files in a local text editor using ``rmate``
+--------------------------------------------------------
+As an alternative to remote-based text editors such as ``vi`` and ``emacs``, we can
+use port-fowarding to set up a `local-based text editor like Atom <https://atom.io>`_ which includes features such as syntax highlighting and code completion. For instructions to install ``rmate`` on your **local machine**, `click here <https://github.com/textmate/rmate>`_. Then to specifically use Atom to edit remote files, `click here <https://atom.io/packages/remote-atom>`_. You will need to add the following line to your ``~/.ssh/config`` file: ``RemoteForward 52698 localhost:52698``.
+
+
+Edit remote python files in a ``jupyter`` environment
+-----------------------------------------------------
+The ``jupyter`` environment is a great environent for data exploration and integrating
```

Comment:
Could also mention that you can send the command by ssh w/o logging to the remote host:
`ssh remotehost "jupyter lab --no-browser --port=3039"`